### PR TITLE
TDEI branding changes

### PR DESCRIPTION
**Pull request by faizm21**

### CODE REVIEW

**Review by sureshgaussian — CHANGES_REQUESTED** [review]
Could you add notes to readme.md on how one can change colors/fonts if necessary and test those locally? UW wants to adjust the colors as they feel they should be using different colors. They will update their design/branding guidelines document as well.

**Comment by faizm21** [comment]
> Could you add notes to readme.md on how one can change colors/fonts if necessary and test those locally? UW wants to adjust the colors as they feel they should be using different colors. They will update their design/branding guidelines document as well.

Readme.md file updated.

File: README.md [hunk 1]
```diff
@@ -25,4 +25,10 @@ Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
 REACT_APP_URL = <User management backend URL>
 REACT_APP_MAP_KEY= <Map API Key for mapbox>
```
Comment (sureshgaussian) [inline README.md:26]: do we still need this?

**Review by sureshgaussian — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (faizm21) [reply README.md]: Removed REACT_APP_MAP_KEY

**Review by sureshgaussian — APPROVED** [review]
lets go ahead and merge

**Pull request merged**